default.nix

```diff
@@ -3,7 +3,22 @@
 with pkgs;
 
 let
-  sources = nix-gitignore.gitignoreSource [ "default.nix" "ci" "tests" "target" ] ./.;
+  # TODO: fix gitignore and replace this by gitignre
+  # See https://github.com/NixOS/nixpkgs/issues/69138
+  sources = builtins.path {
```

Comment:
Hm.. this is really pretty verbose.. Could you perhaps use [cleanSourceWIth](https://github.com/NixOS/nixpkgs/blob/master/lib/sources.nix#L33) ?

Reply:
I'm not sure it would be less verbose since we still have to write the filter. 
Maybe we could use part of the gitinore library but i spent enough time on this issue... 
Moreover, i'm waiting for a fix in gitignore in order to switch back to it.